repositories.md

```diff
@@ -47,10 +47,7 @@ If the Time Server is implemented, the primary is CONDITIONALLY REQUIRED to use
 
 #### ECU Version Report
 
-The ECU version report from each Secondary will contain a token to be sent to the Time Server. To prevent a replay, this token SHOULD be unique for each update cycle. As we expect that these updates will be relatively infrequent (e.g., due to a limited number of write cycles), there will be a sufficient number of tokens to make this possible.
-
-
-The payload of the ECU version report sent to the Director may contain the token sent to the Time Server. This token is part of the version report sent from Secondaries to the Primary. If the token is removed, the signature will not match.
+The ECU version report from each Secondary will contain a token to be sent to the Time Server. To prevent a replay, each token SHOULD be unique per ECU for each update cycle. As we expect that these updates will be relatively infrequent (e.g., due to a limited number of write cycles), there will be a sufficient number of tokens to make this possible. The payload of the ECU version report sent from the Primary to the Director MAY contain the tokens sent to the Time Server. In this case, if any token is removed or changed, the signature will not match.
```

Comment:
```suggestion
The ECU version report from each Secondary will contain a token to be sent to the Time Server in whatever manner the implementer chooses.  For example, the payload of the ECU version report sent from the Primary to the Director MAY contain the tokens sent to the Time Server. In this case, if any token is removed or changed, the signature will not match.  To detect a replay attack, each token SHOULD be unique per ECU. As we expect that these updates will be relatively infrequent (e.g., due to a limited number of write cycles), there will be a sufficient number of tokens to make this possible.  
```

Two changes here.  First, I tried to make it clearer that this was an option, but that sending was required.   Second, I tried to clarify that tokens need to be unique, not just unique per this "cycle" (which seems to mean per update and isn't well defined).

I expect this won't be taken as is, but wanted to use this to try to start more conversation...

Reply:
Actually, this looks great to me. Thanks!